test/unit/presenters/document_list_export_presenter_test.rb

```diff
@@ -69,4 +69,11 @@ class DocumentListExportPresenterTest < ActiveSupport::TestCase
     pub = DocumentListExportPresenter.new(publication)
     assert_equal "force published", pub.state
   end
+
+  test '#unpublished? returns `yes` when a document is unpublished' do
+    unpublished_edition = create(:edition, :unpublished)
+
+    pub = DocumentListExportPresenter.new(unpublished_edition)
```

Comment:
I think `presenter` would be a more appropriate variable name here. `pub` is a bit misleading